Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Install python as a parent image 
+FROM python:3.7
+
+# Set working directory to app
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+COPY . /app
+
+# Install any needed packages specified in requirements.txt
+# TODO: potentially remove 
+RUN pip install -r requirements.txt
+
+# Make port 80 available to the world outside this container
```

Comment:
this isn't exactly what `EXPOSE` does

Reply:
Thanks, I've updated to clarify what EXPOSE does